pkg/api/arc.yaml

```diff
@@ -609,6 +627,11 @@ components:
           description: Transaction status
           example: "ACCEPTED_BY_NETWORK"
           nullable: false
+        callbacks:
```

Comment:
I don't think it is needed to add this information to the api response

Reply:
It's just to show what callback was provided in the request.

Reply:
I think we shouldn't add this field unless somebody asks for it.
Once something is in the api response, it is not simple to remove it again